packages/components/src/components/hds/app-side-nav/index.hbs

```diff
@@ -27,7 +27,7 @@
       />
     {{/if}}
 
-    <div class="hds-app-side-nav__wrapper-body">
+    <div class="hds-app-side-nav__wrapper-body hds-app-side-nav-hide-when-minimized">
```

Comment:
Note: I add to add this class because this is what the component is checking to decide what to add `inert` to. Previously, this was only getting added to the target for the portal ([link](https://github.com/hashicorp/design-system/blob/589e32fadd810ea7dabacb31aef99dc42d76ebd5/packages/components/src/components/hds/app-side-nav/portal/target.hbs#L7)).

Now it gets added when the consumer isn't using Portals too.

Reply:
This seems a little confusing since the 2nd class is always present. Could you just use the existing `hds-app-side-nav__wrapper-body` class name instead?

Reply:
(Also BEM format would be "hds-app-side-nav--hide-when-minimized")

Reply:
Yeah.. I think this comes from SideNav. I can update it to be BEM format.

When the AppSideNav is inserted, it looks for elements with this class name [to decide where to put inert](https://github.com/hashicorp/design-system/blob/130b66858cdf06df6cf7c707e4ac3890dfe7ad99/packages/components/src/components/hds/app-side-nav/index.ts#L184).

I didn't want to change how it works for Portals, which is why I kept using the existing class.

Reply:
It still seems a bit confusing to add an extra class that's always there instead of just using the existing class. Would be curious to know what other engineers think though. @hashicorp/hds-engineering 

Maybe you could at least add a comment to explain why the extra class is there?

Reply:
I'll have a look tomorrow